packages/preact/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "@prairielearn/preact",
+  "version": "1.0.0",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PrairieLearn/PrairieLearn.git",
+    "directory": "packages/preact"
+  },
+  "exports": {
+    ".": "./dist/index.js",
+    "./client": "./dist/client.js",
+    "./hydrated-component": "./dist/hydrated-component/index.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch --preserveWatchOutput & tscp --watch"
```

`tscp` isn't needed here:

```suggestion
    "dev": "tsc --watch --preserveWatchOutput"
```